cli/command/service/opts.go

```diff
@@ -101,9 +101,9 @@ func (i *Uint64Opt) Set(s string) error {
 	return err
 }
 
-// Type returns the type of this option
+// Type returns the type of this option, which will be displayed in `--help` output
 func (i *Uint64Opt) Type() string {
-	return "uint64-ptr"
+	return "uint64"
```

Comment:
I wonder if `uint64` is too much information already; `integer` perhaps (if we give a suitable error message for negative values)

Open to suggestions though /cc @dnephin @bfirsh @vdemeester